cmd/aws-s3-provisioner.go

```diff
@@ -45,8 +45,8 @@ import (
 
 const (
 	provisionerName	 = "aws-s3.io/bucket"
-	awsAccessKeyName = "AWS_ACCESS_KEY_ID"
-	awsSecretKeyName = "AWS_SECRET_ACCESS_KEY"
+	awsAccessKeyName = bktV1.AwsKeyField
```

Comment:
Do we even need these as constants since they are provided by the lib as constants v1alpha1?

Reply:
we don't need them. I added this sort of to remind us that we can use the constants defined in the lib. I'll remove them.